source/ftrack_connect_pipeline_3dsmax/constants/asset/__init__.py

```diff
@@ -1,7 +1,9 @@
 # :coding: utf-8
-# :copyright: Copyright (c) 2014-2020 ftrack
+# :copyright: Copyright (c) 2014-2022 ftrack
 
-FTRACK_PLUGIN_ID = (0x5c8d275e, 0x677d591c)
+FTRACK_PLUGIN_ID = (0x5C8D275E, 0x677D591C)
```

Comment:
Are you sure that capital laters work in here?

Reply:
Plugin object will be attended in a future PR